src/js/index.js

```diff
@@ -37,5 +39,11 @@ export async function init (flags) {
 	initDataStore(user);
 	initIconify(user);
 	initDownloadModal(user);
+
+	let daysLeft = daysUntilMaintenance('October 5, 2021 10:30:00');
```

Comment:
[Question] Does this account for time zones?

Reply:
this is in UTC by default! it's convenient London is in UTC!